sdk/attestation/attestation/src/attestationClient.ts

```diff
@@ -99,17 +142,114 @@ export class AttestationClient {
     // Legacy compatibility classes functions which will be removed eventually.
     this.policy = new Policy(this);
     this.policyCertificates = new PolicyCertificates(this);
-    this.attestation = new Attestation(this);
   }
 
   /**
+   * @ignore
+   * @hidden
```

Comment:
We do not use ignore anymore. I would also say let's not tag this as hidden so that we do not forget to make it private eventually.